docs/third-party-packages.md

```diff
@@ -3,7 +3,6 @@ Starlette has a rapidly growing community of developers, building tools that int
 
 Here are some of those third party packages:
 
-
```

Comment:
Should we maybe be using a markdown linter / formatter to control newlines and stuff like that?

Reply:
I think it would be great if we find one, otherwise we should just do it in PR review.

Reply:
+1 for just doing it in PR review.
Not a big deal either ways, and it's not worth the extra tooling.